README.md

```diff
@@ -111,6 +113,10 @@ cloned.profile.name
 ```
 
 Take a look at our [documentation](https://clowne.evilmartians.io) for more info!
+### Belongs To association
```

Comment:
Can you move this paragraph into the main docs? (here: [include_association](https://github.com/palkan/clowne/blob/master/docs/include_association.md))

Also update `### Supported ORM adapters` table:

Adapter                                   |1:1         | *:1   |  1:M         | M:M                     |
------------------------------------------|------------|-----------|-------------|-------------------------|
[Active Record](https://clowne.evilmartians.io/clowne/docs/active_record.html)  | has_one    | belongs_to   |    has_many    | has_and_belongs_to|
[Sequel](https://clowne.evilmartians.io/clowne/docs/sequel.html)                | one_to_one | - | one_to_many | many_to_many     |

p/s/ And personal request: could you copy\paste this `Supported ORM adapters` table to `include_association` docs (in top) and call it `Supported Associations`? We missed that moment. Thanks!